plugins/module_utils/network/vyos/config/firewall_rules/firewall_rules.py

```diff
@@ -825,29 +842,29 @@ def _add_src_or_dest(self, attr, w, h, cmd, opr=True):
                             )
         return commands
 
-    def search_r_sets_in_have(self, have, w_name, type="rule_sets"):
+    def search_r_sets_in_target(self, target, w_name, type="rule_sets"):
```

Comment:
Why is this change needed? I see the change in the name of the function and arguments. Am I missing something?

Reply:
I tried to explain this change in the commit message where I made it (3afa41de0666def812e275e4941a696eca999542), but GitHub doesn't make those super discoverable, alas.

The change was largely because I kept getting very confused because methods that would call this would pass in their have or their want as the `have` for this method, and the swapping was making it hard to reason about.  I choose `target` because it is independent of `have` and `want`.

Reply:
I understand your reason for this change. But we prefer to have it the way it was because we follow a similar naming convention across all our modules in the vyos collection.